root/etc/e-smith/templates/httpd/proxy-extra/20forcessl_redirect

```diff
@@ -1,6 +1,6 @@
 {
     $OUT = '';
-    if ($HTTP eq 'no' || $Target =~ 'https') {
+    if ($HTTP eq 'no' || $Target =~ 'https://') {
```

Comment:
I think the client redirect to https does not depend on the target URL scheme at all

Reply:
Except if you try to reach a HTTPS proxy from a 80 connection you get a 502 bad gateway. Hence this why to force the redirection when you set a ssl proxy url

Reply:
Agreed. But please set the regexp to match the beginning of `$Target`. In other words:

    $Target =~ m(^https://)